src/core/frames/frame_controller.ts

```diff
@@ -20,6 +20,7 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
   readonly formInterceptor: FormInterceptor
   loadingURL?: string
   formSubmission?: FormSubmission
+  private loadSourceURLWhenChanged = true
```

Comment:
Let's name the property after the state it represents rather than the behavior it triggers.